n3fit/src/evolven3fit_new/evolve.py

```diff
@@ -1,27 +1,27 @@
-import pathlib
 import logging
+import pathlib
 import sys
 
+import eko
 import numpy as np
```

Comment:
```suggestion
import numpy as np
import eko
```

just to put all the eko imports together.

Reply:
@scarlehoff this in general is denied by `isort`, since it is standardizing imports imposing alphabetical sorting.

However, I don't care so much in general, we only use `isort` to avoid imports changing over and over.